digits/model/tasks/caffe_train.py

```diff
@@ -864,11 +864,11 @@ def _pycaffe_args(self, gpu_id):
             gpu_script = "caffe.set_device({id});caffe.set_mode_gpu();".format(id=gpu_id)
         else:
             gpu_script = "caffe.set_mode_cpu();"
+        loading_script = ""
         if self.pretrained_model:
-            weight_files = ','.join(map(lambda x: self.path(x), self.pretrained_model.split(os.path.pathsep)))
-            loading_script = "solv.net.copy_from('{weight}');".format(weight=weight_files)
-        else:
-            loading_script = ""
+            weight_files = map(lambda x: self.path(x), self.pretrained_model.split(os.path.pathsep))
```

Comment:
Should we leave the `if self.pretrained_model` statement in? What if it's `None`?